homeassistant/components/govee_light_local/__init__.py

```diff
@@ -14,18 +14,19 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.exceptions import ConfigEntryNotReady
 
-from .const import DISCOVERY_TIMEOUT, DOMAIN
+from .const import DISCOVERY_TIMEOUT
 from .coordinator import GoveeLocalApiCoordinator
 
 PLATFORMS: list[Platform] = [Platform.LIGHT]
 
 _LOGGER = logging.getLogger(__name__)
 
+type GoveeLocalConfigEntry = ConfigEntry[GoveeLocalApiCoordinator]
```

Comment:
It is tighly linked to the coordinator, so I suggest moving this to `coordinator.py`
It will also help to avoid circular reference as we look to add the config entry explicitely to the coordinator initialisation.

Reply:
Makes sense 🙂